scripts/bump-version.sh

```diff
@@ -34,10 +34,10 @@ echo "* Bumping version in Dockerfile.rhel"
 sed -i "s/\(version=\)[0-9]*\.[0-9]*\.[0-9]*/\1${NEW_VERSION}/g" Dockerfile.rhel
 check_version Dockerfile.rhel
 
-echo "****************************************************************************************"
-echo "* Don't forget to update homebrew package at https://github.com/kadel/homebrew-odo ! *"
-echo "****************************************************************************************"
+echo "* Bumping version in build/VERSION"
+sed -i "s/\(version=\)[0-9]*\.[0-9]*\.[0-9]*/\1${NEW_VERSION}/g" build/VERSION
+check_version build/VERSION
 
 echo "****************************************************************************************"
-echo "* Don't forget to update build/VERSION once the binaries become available !            *"
-echo "****************************************************************************************"
+echo "* Don't forget to update homebrew package at https://github.com/kadel/homebrew-odo ! *"
```

Comment:
Can we not automate this too? Maybe use bot account

Reply:
I don't know as I have no clue about things on homebrew side. @kadel can you help with this one?

Reply:
I'm working on adding odo to homebrew core repo so we don't have this separate Tab. Then I will revisit the automation.